src/mixxx.cpp

```diff
@@ -1503,14 +1502,20 @@ void MixxxMainWindow::rebootMixxxView() {
         int newX = initPosition.x() + (initSize.width() - m_pWidgetParent->width()) / 2;
         int newY = initPosition.y() + (initSize.height() - m_pWidgetParent->height()) / 2;
 
-        const QScreen* primaryScreen = getPrimaryScreen();
-        if (primaryScreen) {
-            newX = std::max(0, std::min(newX, primaryScreen->geometry().width() - m_pWidgetParent->width()));
-            newY = std::max(0, std::min(newY, primaryScreen->geometry().height() - m_pWidgetParent->height()));
-            move(newX,newY);
-        } else {
+        const QScreen* const pScreen = mixxx::widgethelper::getScreen(*this);
+        VERIFY_OR_DEBUG_ASSERT(pScreen) {
             qWarning() << "Unable to move window inside screen borders.";
         }
+        else {
+            newX = std::max(0,
+                    std::min(newX,
+                            pScreen->geometry().width() -
+                                    m_pWidgetParent->width()));
```

Comment:
Can we move this into a temporary const variable? Then line becomes shorter without these ugly linebreaks and the code might become easier to grasp.